src/themes/hugo-theme-learn/assets/sass/_theme.scss

```diff
@@ -543,6 +551,13 @@ pre {
   }
 }
 
+div.highlight {
```

Comment:
No chance that `highlight` class could be on another element type than a `div`?

Reply:
Not impossible, but this targets precisely the div that is added by Hugo around code blocks